dhcpv4/nclient4/example_lease_test.go

```diff
@@ -0,0 +1,40 @@
+//this is an example for nclient4 with lease
+
+package nclient4_test
+
+import (
+	"context"
+	"log"
+
+	"github.com/insomniacslk/dhcp/dhcpv4"
+	"github.com/insomniacslk/dhcp/dhcpv4/nclient4"
+)
+
+func Example_dHCPv4ClientLease() {
+	ifname := "eth0"
+	remoteID := "client-1"
+	var idoptlist dhcpv4.OptionCodeList
+	//specify option82 is part of client identification used by DHCPv4 server
```

Comment:
As above, I wouldn't use the the relay agent info in the client code.

Reply:
see my comment above